DataStructure/Queue/queue.java

```diff
@@ -0,0 +1,192 @@
+package Queue;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Queue<V> {
+
+	class Node {
+		V data;
+		Node next;
+
+		Node(V val) {
+			this.data = val;
+			this.next = null;
+		}
+	}
+
+	private Node front;
+	private Node rear;
+	private int size;
+
+	// Constructor calling of class queue
+	public Queue() {
+		front = null;
+		rear = null;
+		size = 0;
+	}
+
+	// Enqueue operation in queue
+	public void push(V data) {
+		if (front == null) {
+			Node newnode = new Node(data);
+			front = rear = newnode;
+			size++;
+			return;
+		} else {
+			Node newnode = new Node(data);
+			rear.next = newnode;
+			rear = rear.next;
+			size++;
+
+		}
+
+	}
+
+	// dequeue operation in queue
+	public void pop() {
+		if (front == null) {
+			return;
+		}
+		front = front.next;
+		size--;
+
+	}
+
+	// get the front Value from queue
+	public V Front() {
```

Comment:
Function naming should also be in Camel Casing like Front should be front. Change  this throughout the modules.